prac_05/colour_codes.py

```diff
@@ -0,0 +1,13 @@
+COLOUR_CODES = {"Aliceblue": "#f0f8ff", "Blanchedalmond": "#ffebcd", "Blueviolet": "#8a2be2", "Cadetblue": "#5f9ea0",
+                "Blue": "#0000ff", "Cyan": "00ffff", "Darkgreen": "#006400", "Dodgerblue": "#1e90ff"}
+
+colour = input("Enter colour name: ").capitalize()
```

Comment:
careful with .capatalize as the hex color names are not all in that format. you have selected names that work, but not all hex color names start with a capital and many include a number which could break your code, i would recommend no modifier on the input of the color names